app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= stylesheet_link_tag "application"%>
+
+<div class="logger_in_form">
+<p><%= notice %></p>
+
+<%= render @recipe %>
+
+<div>
+  <%= link_to 'Back', recipes_path %>
+  <%= link_to 'Edit', edit_recipe_path(@recipe) %>
+  <%= link_to 'Destroy', @recipe, method: :delete %>
+</div>
```

Comment:
- [ ] Please add buttons or links for adding ingredients and generating a shopping list.

- [ ] Same here, Kindly make this view matches the given wireframe.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/209049483-26e79c34-40cd-4405-84b8-b65b282135ba.png)  | ![image](https://user-images.githubusercontent.com/68796232/209049616-3db7935c-2285-4715-90bb-678fb392495b.png)  |

Reply:
thank you for this